### DISCUSSION TITLE
Exporting Alerts to .csv or similar

### DISCUSSION THREAD

**Discussion by juddcbrown** [Security-Onion-Solutions/securityonion#4907] [Unsupported Versions] [answered]
Is there a way to export all Alerts to another file for review?

Thanks!

**Comment by joeb1kenobe** [comment, marked as answer]
Bring up the alerts in kibana. You should be able to export from there.

Joe

On Thu, Jul 22, 2021 at 7:06 PM juddcbrown ***@***.***> wrote:

> Is there a way to export all Alerts to another file for review?
>
> Thanks!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/4907>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAD6W6WV7PKRNRY2MHMZOGTTZCP7NANCNFSM5A26DHOQ>
> .
>
-- 
Joe Brown

**Comment by juddcbrown** [comment]
Thanks Joe.

On Thu, Jul 22, 2021 at 6:45 PM Joe Brown ***@***.***> wrote:

> Bring up the alerts in kibana. You should be able to export from there.
>
> Joe
>
> On Thu, Jul 22, 2021 at 7:06 PM juddcbrown ***@***.***> wrote:
>
> > Is there a way to export all Alerts to another file for review?
> >
> > Thanks!
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/Security-Onion-Solutions/securityonion/discussions/4907
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAD6W6WV7PKRNRY2MHMZOGTTZCP7NANCNFSM5A26DHOQ
> >
> > .
> >
> --
> Joe Brown
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/4907#discussioncomment-1039942>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH4XT7TZOL7URGACVNKXH5LTZCUSVANCNFSM5A26DHOQ>
> .
>


-- 
Judd Brown
